src/renderer/services/validation/schemas/index.ts

```diff
@@ -0,0 +1,22 @@
+export {
+    schema as KeySchema,
+    KEY_START_WITH_PREFIX_MESSAGE,
+    KEY_WRONG_CHARACTERS_MESSAGE,
+    KEY_NOT_STRING_MESSAGE
+} from "./key";
+
+export {
+    schema as PrivateKeySchema,
+    PRIVATE_KEY_WRONG_LENGTH_MESSAGE
+} from "./private-key";
+
+export {
+    schema as PublicKeySchema,
+    PUBLIC_KEY_WRONG_LENGTH_MESSAGE
+} from "./public-key";
+
+export {
+    schema as MnemonicSchema,
```

Comment:
rename those things in their files. index should just have
`export * from "./mnemonic`

same thing goes for other stuff